extensions/resteasy-reactive/quarkus-jaxrs-client/deployment/src/main/java/io/quarkus/resteasy/reactive/client/deployment/JaxrsClientProcessor.java

```diff
@@ -241,12 +248,37 @@ public void registerInvocationCallbacks(CombinedIndexBuildItem index, ResteasyRe
                     }
                     //TODO: async return types
 
-                    ResultHandle result = m
-                            .invokeInterfaceMethod(
-                                    MethodDescriptor.ofMethod(Invocation.Builder.class, "method", Object.class, String.class,
-                                            Class.class),
-                                    builder, m.load(method.getHttpMethod()), m.loadClass(method.getSimpleReturnType()));
-
+                    ResultHandle result;
+                    if (bodyParameterIdx != null) {
+                        String mediaTypeValue = "application/octet-stream";
```

Comment:
This should use the constant javax.ws.rs.core.MediaType#APPLICATION_OCTET_STREAM